### PR TITLE
Extra Variables added for INCORE specific

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: charts/datawolf/templates/deployment.yaml [hunk 1]
```diff
@@ -94,6 +94,16 @@ spec:
               value: {{ .Values.jobs.cpu | quote }}
             - name: KUBERNETES_MEMORY
               value: {{ .Values.jobs.memory | quote }}
+            - name: DATASET_DAO
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply charts/datawolf/templates/deployment.yaml:97]: > {{ if .Values.extraEnvVars -}}
>     {{ .Values.extraEnvVars | toYaml | nindent 12 }}
>     {{ end -}}

does above work? It didn't create -name, value pair so I made it like
            {{- if .Values.extraEnvVars }}
            {{- range $key, $value := .Values.extraEnvVars }}
            - name: {{ $key | upper }}
              value: {{ $value | quote }}
            {{- end }}
            {{- end }}
            
so we can just simply put like

extraEnvVars:
  INCORE_USER: incore_user
  
  in values.yaml

File: charts/datawolf/values.yaml [hunk 2]
```diff
@@ -33,6 +33,13 @@ jobs:
   # default memory in GB per job
   memory: 4.0
 
+extraEnvVars:
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply charts/datawolf/values.yaml:36]: updated

File: charts/datawolf/templates/deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ywkim312) [reply charts/datawolf/templates/deployment.yaml]: > next in the in-core helm chart we can add to the datawolf section the variables.

I will work on it

[the remark this answers is not in the document]
Comment (ywkim312) [reply charts/datawolf/templates/deployment.yaml]: > {{ if .Values.extraEnvVars -}}
>     {{ .Values.extraEnvVars | toYaml | nindent 12 }}
>     {{ end -}}

Never mind, I changed it your way. I will make the incore helm to work with this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 - User information to environment of process running the tool [#28](https://github.com/ncsa/datawolf/issues/28)
+- Extra environment variables for IN-CORE specific [#35](https://github.com/ncsa/datawolf/issues/35)
```
Comment (robkooper) [inline CHANGELOG.md:11]:
```suggestion
- Ability for extra environment variables (used by IN-CORE) [#35](https://github.com/ncsa/datawolf/issues/35)
```

**Review by robkooper — CHANGES_REQUESTED** [review]
small tweak of changelog

File: CHANGELOG.md (hunk 3 above)
Comment (ywkim312) [reply CHANGELOG.md]: Updated as suggested

**Comment by ywkim312** [comment]
> small tweak of changelog

implemented

**Review by robkooper — APPROVED** [review]

**Review by navarroc — APPROVED** [review]

**Pull request merged**